api-2.0/pom.xml

```diff
@@ -16,7 +16,6 @@
 	<properties>
 		<openmrs.2.0.api.version>2.0.0-alpha</openmrs.2.0.api.version>
 	</properties>
-
```

Comment:
I don't think we need this change in this PR?

Reply:
Yes i wanted to change it at first , in my mind i was thinking of upgrading it to the current version of openmrs core, but according  to what @dkayiwa  said, this it ok